src/sentry/static/sentry/app/views/settings/organizationIntegrations/configureIntegration.tsx

```diff
@@ -68,11 +82,11 @@ class ConfigureIntegration extends AsyncView {
           </Form>
         )}
 
-        {provider.features.includes('alert-rule') && (
+        {provider && provider.features.includes('alert-rule') && (
           <IntegrationAlertRules integration={integration} />
         )}
 
-        {provider.features.includes('commits') && (
+        {provider && provider.features.includes('commits') && (
```

Comment:
@nolachen How come we need to add this new check?

Reply:
@scefali provider comes from this line:

```
const provider = this.state.config.providers.find(
      p => p.key === integration.provider.key
    );
```

which means it's possible that it's undefined

Reply:
@nolachen That makes sense. TS was complaining even though in reality there's always a matching provider.